.github/workflows/sycl-windows-build.yml

```diff
@@ -182,3 +189,10 @@ jobs:
         name: sycl_windows_default
         path: ${{ inputs.artifact_archive_name }}
         retention-days: ${{ inputs.retention-days }}
+    - name:  Detect hung tests
+      if: always()
+      shell: powershell
+      run: |
+        Invoke-WebRequest -Uri "https://raw.githubusercontent.com/intel/llvm/refs/heads/sycl/devops/scripts/windows_detect_hung_tests.ps1" -OutFile "windows_detect_hung_tests.ps1"      
+        powershell.exe -File windows_detect_hung_tests.ps1
```

Comment:
Should we delete the script after executing it?

Reply:
better safe than sorry, will add thx